src/Microsoft.Tye.Hosting/TransformProjectsIntoContainers.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Tye.Hosting.Model;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Tye.Hosting
+{
+    public class TransformProjectsIntoContainers : IApplicationProcessor
+    {
+        private readonly ILogger _logger;
+
+        public TransformProjectsIntoContainers(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public async Task StartAsync(Model.Application application)
+        {
+            // This transforms a ProjectRunInfo into
+
+            foreach (var s in application.Services.Values)
+            {
+                if (s.Description.RunInfo is ProjectRunInfo project)
+                {
+                    await TransformProjectToContainer(application, s, project);
```

Comment:
This builds the projects and can be done in parallel (like the process runner currently does)

Reply:
But it isn't in parallel today, correct? Why can't you do a Task.Run on each and await all of them at the end?

Reply:
I copied what the other processors do. Not quite Task.Run but it goes parallel if the build finishes asynchronously.